test/functional/currency/currency-formatter.js

```diff
@@ -45,7 +45,9 @@ QUnit.module( ".currencyFormatter( currency [, options] )", {
 		});
 		Globalize.locale( "en" );
 	},
-	teardown: util.resetCldrContent
+	teardown: function() {
+		util.resetCldrContent();
```

Comment:
Why this change? The function doesn't seem to care in which scope it is called. At least keep it out of this PR? 17 less files changed is worth it.